Makefile-rpm-ostree.am

```diff
@@ -88,11 +88,16 @@ endif
 
 if ENABLE_RUST
 librpmostree_rust_path = @abs_top_builddir@/target/@RUST_TARGET_SUBDIR@/librpmostree_rust.a
+# If the target directory exists, use --frozen; we don't
+# want to (by default) touch the Internet during builds here.
+# One reason is that we want to support `make && sudo make install`,
+# and we don't want to do `sudo cargo` there.
```

Comment:
Hmm, though is that just a symptom? Why are we running `cargo` again at `make install` time in the first place? I think this is what I was hitting in https://github.com/ostreedev/ostree/pull/1619 (and do still hit it from time to time in rpm-ostree). Then I have to `rm -rf target/` to let it build again.

Anyway, this patch makes sense regardless!

Reply:
Yeah...I'm not entirely sure honestly.